__tests__/integration/schemaValidation.test.js

```diff
@@ -3,14 +3,14 @@ import pupExpect from 'expect-puppeteer'
 
 describe('Editor', () => {
   beforeAll(async () => {
-    await page.goto('http://127.0.0.1:8080/editor')
+    await page.goto('http://127.0.0.1:8080/import')
   })
 
   describe('importing a profile/template', () => {
     beforeAll(async() => {
       page.waitForSelector('button#ImportProfile')
-      await page.click('button#ImportProfile')
-      await pupExpect(page).toMatch('Loaded resource template file')
+      // await page.click('button#ImportProfile')
+      // await pupExpect(page).toMatch('Loaded resource template file')
```

Comment:
CAN we now import a profile?  if we do, is it schema validated?

Reply:
Yes, I think we want to keep these here because we'll be using these lines to test the **Import Resource Tab** loading and validating, we may need to match on a different selector however.